pkg/daemon/osrelease.go

```diff
@@ -22,7 +25,16 @@ func (os OperatingSystem) IsRHCOS() bool {
 	return os.ID == "rhcos"
 }
 
-// IsFCOS is true if the OS is RHEL CoreOS
+// IsRHCOS9 is true if the OS is RHEL CoreOS 9
+func (os OperatingSystem) IsRHCOS9() bool {
```

Comment:
Do we know if we have infra to test this today in RHCOS9? Maybe we can track this in a card and implement along with test once we have OCP based on RHCOS 9?

Reply:
Not sure, but I agree that it'd be better to hold off on the RHCOS9 part until that's testable. For OKD it'd be good to get this in already, though.